client/webserver/site/src/js/markets.js

```diff
@@ -2340,8 +2340,13 @@ function makeMarket (host, base, quote) {
 /* marketID creates a DEX-compatible market name from the ticker symbols. */
 export function marketID (b, q) { return `${b}_${q}` }
 
-/* convertConventional converts the float string to atoms. */
+/* convertConventional converts the float string to conventional units. */
 function convertConventional (s, conversionFactor) {
+  return Math.round(parseFloat(s) / conversionFactor)
```

Comment:
I don't think we want a `round` after dividing to get conventional units.

Reply:
Should have been floor.

Reply:
Wait, we want decimals in conventional units...

Reply:
floor is right right?

Reply:
Don't think so.  Say there was a market with a lot size of 5000000 atoms (0.5 DCR).  This quantity field would need decimals

Reply:
I think there's some unnecessary back and forth conversions, but it works right.

![image](https://user-images.githubusercontent.com/9373513/152436789-17c96419-8727-4948-aaa3-5c28214ef5ae.png)

![image](https://user-images.githubusercontent.com/9373513/152436836-84d03820-4b1d-4d7a-8b35-a0a954ac6644.png)

So I kinda thought this would just be `return parseFloat(s) * conversionFactor`.
If we find we need to deal with floating point precision issues, we can do all this string->float->string business, but I think the precision would not be from the unitInfo's conversion factor but rather lot size in atoms.

Reply:
Yeah, `return parseFloat(s) / conversionFactor` is fine right? Won't it always be 1000000 or something like that? I don't know why I would expect something different.